src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -189,12 +196,29 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
 
     file_stat->size = (handle->ffd.nFileSizeHigh * ((int64_t)(MAXDWORD)+1)) + handle->ffd.nFileSizeLow;
     file_stat->modtime = FileTimeToUnixTime(&handle->ffd.ftLastWriteTime, &rem);
+
+    std::wstring fullPath;
+    fullPath.reserve(std::wcslen(handle->path) + std::wcslen(handle->ffd.cFileName));
+    fullPath += handle->path;
+    fullPath += handle->ffd.cFileName;
+
+    if(_csync_vio_local_stat_mb(fullPath.data(), file_stat.get()) < 0) {
+         file_stat->type = CSYNC_FTW_TYPE_SKIP;
+    }
+
     return file_stat;
 }
 
 
+int csync_vio_local_stat(const char *uri, csync_file_stat_t *buf)
+{
+    mbchar_t *wuri = c_utf8_path_to_locale( uri );
+    int rc = _csync_vio_local_stat_mb(wuri, buf);
```

Comment:
The linux version does ` *buf = csync_file_stat_t();` beforehand

Reply:
I did not change the behaivor.

Reply:
The interface being different between linux and windows variants of this function should be a bug. Either clearing `*buf` is pointless, or this function incorrectly doesn't do it. But yes, it's independent of your change.